Dosbodoke/chooselife
# :sparkles: Revert pick location

## What is the purpose of this PR
Add functionality to reset the picked location without needing to go back to home screen and click **+** again

## What was done to achieve
Added a button that remove the last item of `markers` array on clicked

## How to test if it works?
Click on the `+` sign to create a new Highline
Place the anchor A
Check if the new button appears
Click the new button and check if it disappears

Place the anchor's A and B, click the reset button, and check if anchor B disappears, click again and check if all anchors disappear.
